### PR TITLE
Expose an option to disable cookies

### LINKED ISSUE #335: option to disable cookies completely
In order to use AI with a customer, we need to remove all cookies. There is already `Util.canUseCookies()`, could this function take an global option to disable it? Same for localstorage.

### PR DESCRIPTION
**Pull request by kamilszostak**
Fixes #335

### CODE REVIEW

File: API-reference.md [hunk 1]
```diff
@@ -281,6 +281,10 @@ Values that control how the telemetry data is sent.
         // The url from where the JS SDK will be downloaded. 
         // Default 'https://az416426.vo.msecnd.net/scripts/a/ai.0.js'
         url: string;
+
+        // If true, the SDK will not store or read any data from cookies.
+        // Default: false
+        disableCookies: boolean;
```
Comment (SergeyKanzhelev) [inline API-reference.md]: can you move it closer to `cookieDomain` setting so they will be sitting together.

**Comment by pelikhan** [comment]
Can there be a flag to disable local storage as well?

File: API-reference.md (hunk 1 above)
Comment (kamilszostak) [inline API-reference.md]: Done

**Comment by kamilszostak** [comment]
I updated the pull request. 

* Exposed and option to disable session and local storage
* CanUseLocal and CanUseSession storage methods now cache the results, so it will be slightly faster
* Renamed configuration fields : `isCookieUseDisabled` and `isStorageUseDisabled`

**Comment by pelikhan** [comment]
Looks great!

**Pull request merged**